server/scripts/fine_tune.py

```diff
@@ -0,0 +1,72 @@
+import os
+import requests
+import json
+from tqdm import tqdm
+from dotenv import load_dotenv
+from openai import OpenAI
+import random
+from utilities.constants.script_constants import(
+    GENERATE_BATCH_SCRIPT_PATH,
+    SAMPLE_QUESTIONS_DIR,
+    FINE_TUNE_NUMBER,
+    FINE_TUNE_SYSTEM_MESSAGE,
+    FINE_TUNE_EXAMPLES_DIR,
+    FINE_TUNE_EXAMPLES_FILE_PREFIX,
+    GENERATE_BATCH_RELATIVE_PATH,
+    DB_CHANGE_ENPOINT,
+    PROMPT_GENERATE_ENDPOINT,
+    PROMPT_TYPE,
+    NUM_SHOTS
+)
+
+# getting databases 
+directories = [d for d in os.listdir(GENERATE_BATCH_SCRIPT_PATH) if os.path.isdir(os.path.join(GENERATE_BATCH_SCRIPT_PATH, d))]
+
+ft_examples=[]
+
+# iterating over all Databases to generate prompts and make batch input files
+for database in tqdm(directories,desc=f'Processing Directories:'):
+
+    response=requests.post(DB_CHANGE_ENPOINT,json={'database_type':database,"sample_path":f"{GENERATE_BATCH_RELATIVE_PATH}{database}{SAMPLE_QUESTIONS_DIR}unmasked_{database}.json"})
+
+    with open(f"{GENERATE_BATCH_SCRIPT_PATH}{database}{SAMPLE_QUESTIONS_DIR}unmasked_{database}.json") as file:
+        sample_qs=json.loads(file.read())
+        file.close()
+    
+    if not response.status_code==200:
```

Comment:
Always use intermediate strings and number for the checks as constants declared at the top of the script